L1Trigger/L1THGCal/src/concentrator/HGCalConcentratorSuperTriggerCellImpl.cc

```diff
@@ -44,10 +44,9 @@ void HGCalConcentratorSuperTriggerCellImpl::createAllTriggerCells(
     } else if (triggerTools_.isScintillator(output_ids.at(0))) {
       thickness = HGCalTriggerTools::kScintillatorPseudoThicknessIndex_;
     }
-
     for (const auto& id : output_ids) {
-      if (fixedDataSizePerHGCROC_ && thickness > kHighDensityThickness_ &&
-          id != coarseTCmapping_.getRepresentativeDetId(id)) {
+      if ( ((fixedDataSizePerHGCROC_ && thickness > kHighDensityThickness_)|| coarsenTriggerCells_) &&
```

Comment:
Does this fix have a an impact on some options that have been used before?

Reply:
It should only affect the case where normal TCs are coarsened, and then passed to the STC (apart from the `_fixedDataSizePerHGCROC' which is anyway treated correctly).
I don't think we ever previously saw results like this (other than what I showed last week, where the fix was already included locally).